apps/avifenc.c

```diff
@@ -417,6 +417,10 @@ static avifBool avifImageSplitGrid(const avifImage * gridSplitImage, uint32_t gr
         for (uint32_t gridX = 0; gridX < gridCols; ++gridX) {
             uint32_t gridIndex = gridX + (gridY * gridCols);
             avifImage * cellImage = avifImageCreateEmpty();
+            if (!cellImage) {
+                fprintf(stderr, "ERROR: Cell creation failed: out-of-memory\n");
```

Comment:
Nit: In error messages like this, "out of memory" doesn't need to be hyphenated.
The hyphenated form "out-of-memory" is used as an adjective, such as "out-of-memory error".